packages/airnode-deployer/README.md

```diff
@@ -91,4 +94,9 @@ Options:
   -c, --cloud-provider         Cloud provider                                                    [choices: "aws", "gcp"]
   -e, --region                 Region                                                                           [string]
   -p, --project-id             Project ID (GCP only)                                                            [string]
+
+Examples:
+  deployer.ts remove -r myOutput/receipt.json
```

Comment:
Btw. this `deployer.ts` is there only because the filename is `deployer.ts` and you probably ran `yarn cli --help`.

Also, I'd probably remove this `### Commands` section. WDYT: @amarthadan ?

Reply:
Thanks, fixed!